x-pack/test/api_integration/deployment_agnostic/configs/serverless/oblt.index.ts

```diff
@@ -23,5 +23,6 @@ export default function ({ loadTestFile }: DeploymentAgnosticFtrProviderContext)
     loadTestFile(require.resolve('../../apis/observability/slo'));
     loadTestFile(require.resolve('../../apis/observability/onboarding'));
     loadTestFile(require.resolve('../../apis/observability/incident_management'));
+    loadTestFile(require.resolve('../../apis/intercepts'));
```

Comment:
Is there a reason why this is only included in observability serverless tests and not for search and security projects?

Reply:
This was an omission... resolved in [`f0b83c3` (#221743)](https://github.com/elastic/kibana/pull/221743/commits/f0b83c34a6550e88f5ce3929a8848bad0462055b)